test/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ if(KOKKOS_DEVICES MATCHES "OpenMP")
   list(APPEND ARBORX_DEVICE_TYPES Kokkos::OpenMP::device_type)
 endif()
 if(KOKKOS_DEVICES MATCHES "Cuda")
-  list(APPEND ARBORX_DEVICE_TYPES Kokkos::CudaUVMSpace::device_type)
+  list(APPEND ARBORX_DEVICE_TYPES Kokkos::Cuda::device_type)
```

Comment:
Is this change temporary or do you want to stop testing for UVM?

Reply:
We will need to test for both eventually.  Wasn't planning on fixing it in this PR

Reply:
We should open an issue about it otherwise we will forget/